azurelinuxagent/common/osutil/ubuntu.py

```diff
@@ -29,15 +29,20 @@ class Ubuntu14OSUtil(DefaultOSUtil):
     def __init__(self):
         super(Ubuntu14OSUtil, self).__init__()
         self.jit_enabled = True
+        self.service_name = self.get_service_name()
+
+    @staticmethod
+    def get_service_name():
+        return "walinuxagent"
```

Comment:
Just a food for thought - Can we define the Service Names (`waagent`, `walinuxagent`) in a place as Constants (at the defaultOSUtil) so that these loose strings are not present?

Similar constants can be used in the test as well - for cleanliness.

Reply:
Similar for the service commands, but thats for later.

Reply:
I agree, but that cleanup should also involve taking the same constants out of `setup.py`, so I'd rather do it separately.